modules/ossm-rn-new-features-1x.adoc

```diff
@@ -19,6 +19,28 @@ Module included in the following assemblies:
 * *Policy Enforcement* - Apply organizational policy to the interaction between services, ensure access policies are enforced and resources are fairly distributed among consumers. Policy changes are made by configuring the mesh, not by changing application code.
 * *Telemetry* - Gain understanding of the dependencies between services and the nature and flow of traffic between them, providing the ability to quickly identify issues.
 
+== New features {SMProductName} 1.1.19
```

Comment:
The version for SM 1.1 is 1.1.18-2.

FYI: 1.1 Errata name for 2.1.3 release: Red Hat OpenShift Service Mesh 1.1.18.2 Containers